src/data-source.js

```diff
@@ -28,13 +28,17 @@ function dataSource(params) {
   }
 
   // expect the _ow_path to start with /https:// or /https%3a%2f%2f
-  if (path.startsWith('/https%3A%2F%2F')) {
-    return new URL(decodeURIComponent(path.substring(1)));
+  // the escaping done by runtime is inconsistent, the : may be decoded
+  if (path.match(/^\/https(:|%3A)%2F/)) {
+    return new URL(decodeURIComponent(path.substring(1)
+      .replace(/^https(:|%3A)2F([^%])/, 'https://$2')));
```

Comment:
isn't here a `%` missing?

```suggestion
      .replace(/^https(:|%3A)%2F([^%])/, 'https://$2')));
```